mmv1/third_party/terraform/services/bigquery/data_source_google_bigquery_tables_test.go

```diff
@@ -0,0 +1,70 @@
+package bigquery_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccDataSourceGoogleBigqueryTables_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckBigQueryTableDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceGoogleBigqueryTables_basic(context),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.google_bigquery_tables.example", "tables.#", "1"),
+					resource.TestCheckResourceAttr("data.google_bigquery_tables.example", "tables.0.table_id", "test_table"),
+					resource.TestCheckResourceAttr("data.google_bigquery_tables.example", "tables.0.labels.%", "1"),
+					resource.TestCheckResourceAttr("data.google_bigquery_tables.example", "tables.0.labels.goog-terraform-provisioned", "true"),
+				),
+			},
+		},
+	})
+}
+
+func testAccDataSourceGoogleBigqueryTables_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+  
+  resource "google_bigquery_dataset" "test" {
+    dataset_id                  = "tf_test_ds_%{random_suffix}"
+    friendly_name               = "testing"
+    description                 = "This is a test description"
+    location                    = "US"
+    default_table_expiration_ms = 3600000
+  }
+
+  resource "google_bigquery_table" "test" {
+    dataset_id        = google_bigquery_dataset.test.dataset_id
+    table_id          = "test_table"
+    deletion_protection = false
+    schema     = <<EOF
+    [
+      {
+        "name": "name",
+        "type": "STRING",
+        "mode": "NULLABLE"
+      }
+    ]
+    EOF
+    # Default label for a tf-provisioned table (referenced in the test suite):
+    # labels = {
+    #   goog-terraform-provisioned = "true"
+    # }
```

Comment:
This can be deleted - the provider automatically adds this label to all resources now

Reply:
Ah yeah this was essentially a contextual comment for anyone new reading the code not figuring out where the test [here](https://github.com/GoogleCloudPlatform/magic-modules/pull/11552/files#diff-45919cfdf826343c45a26d709c331f1a41b9a5170d1f26c9144e55041a7ad5a8R28) is showing up from